src/mvAppItem.cpp

```diff
@@ -2084,8 +2098,20 @@ DearPyGui::GetEntityParser(mvAppItemType type)
         args.push_back({ mvPyDataType::Bool, "menubar", mvArgType::KEYWORD_ARG, "False", "Shows/Hides the menubar at the top." });
         args.push_back({ mvPyDataType::Bool, "no_scroll_with_mouse", mvArgType::KEYWORD_ARG, "False", "Disable user vertically scrolling with mouse wheel." });
         args.push_back({ mvPyDataType::Bool, "flattened_navigation", mvArgType::KEYWORD_ARG, "True", "Allow gamepad/keyboard navigation to cross over parent border to this child (only use on child that have no scrolling!)" });
-
-        setup.about = "Adds an embedded child window. Will show scrollbars when items do not fit.";
+        args.push_back({ mvPyDataType::Bool, "always_use_window_padding", mvArgType::KEYWORD_ARG, "False", "Pad with style.WindowPadding even if no border are drawn (no padding by default for non-bordered child windows because it makes more sense)" });
+        args.push_back({ mvPyDataType::Bool, "resize_x", mvArgType::KEYWORD_ARG, "False", "Allow resize from right border (layout direction). Enable .ini saving (unless 'no_saved_settings' is passed as parameter)" });
```

Comment:
`resize_x`, `resize_y`: To me, `resizable_x/y` sounds a bit better. They convey the fact that the window is resizable, whereas `resize_x` sounds as if we're requesting DPG to resize it.